backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientResolver.java

```diff
@@ -37,13 +37,25 @@ public long patientsCount(UUID facilityId, boolean showDeleted, String namePrefi
 
   public boolean patientExists(
       String firstName, String lastName, LocalDate birthDate, String zipCode, UUID facilityId) {
+    // Backwards compatibility shim -- zipCode is unused
     Organization org = _os.getCurrentOrganization();
     Optional<Facility> facility =
         facilityId == null
             ? Optional.empty()
             : Optional.of(_os.getFacilityInCurrentOrg(facilityId));
 
-    return _ps.isDuplicatePatient(firstName, lastName, birthDate, zipCode, org, facility);
+    return _ps.isDuplicatePatient(firstName, lastName, birthDate, org, facility);
+  }
+
+  public boolean patientExistsNew(
```

Comment:
Could this be renamed to patientExistsWithoutZip or similar?

Reply:
Happy to! The `New` suffix has been a soft convention on the team when slotting in new queries alongside backwards-compatible ones, but it obviously doesn't carry any semantic information. See f49d522.